### DISCUSSION TITLE
generator-jhipster-micronaut future

### DISCUSSION THREAD

**Discussion by vcupelloni** [jhipster/generator-jhipster-micronaut#393] [Q&A]
Hi everyone, maybe I'm going to be a little rough :)
What are the long-term plans for this blueprint? Are there still active mantainers or is it slowly slipping into oblivion? 
I ask because unfortunately I see the open issue picture a bit stagnant and the generator itself has a lot of out-of-the-box problems...
I've also tried to report them by proposing for resolution but haven't received any feedback from anyone.

I want to be clear, mine is not a criticism of the people working on the project, I know very well that everyone has a lot of commitments and running an opensource project is very challenging, I just want to know if people still believe in this generator.

**Comment by vcupelloni** [comment]
I think the lack of answers is in itself an answer..... :(

**Reply by mraible** [reply]
Yes, you are correct. This blueprint isn't very active because no one is contributing to it. It was originally created by folks from Object Computing Inc. and they haven't been very active in maintaining it. I think this is mostly because it was [created several versions ago](https://micronaut.io/2021/04/19/jhipster-micronaut-blueprint-1-0-release/) (for JHipster 6) and it's been difficult to update it between major versions w/o core contributor's help. 

I also think the activity is down because the popularity of Micronaut is waning. The link below is to a slide where I compared GitHub star growth for Spring Boot, Quarkus, Micronaut, and Helidon. Of these four, you can see that Micronaut is the only one that's trending down. 

https://speakerdeck.com/mraible/reactive-java-rest-apis-ftw-virtual-jug-2024?slide=55

For me, personally, I'm not contributing much lately because I got a new job and it has no relation to JHipster, unlike my previous one.

**Comment by mshima** [comment]
We’ve did a lot of work to integrate micronaut blueprint with generator-jhipster recently.
Micronaut releases will happen after a new generator-jhipster is integrated, expect more activity near a release.

What to expect:
- updated ui
- improvements to common stack like docker compose files
- some maven and gradle build stack like jib, frontend:
https://github.com/jhipster/generator-jhipster/tree/main/generators/java/generators

But don't expect many micronaut specific changes since we are laking active contributors to it.
